docs/developer-guide/build-service.md

```diff
@@ -1,101 +1,174 @@
 
-If building modules sounds like a lot of work, you can let someone else do it for you and use our **build service**, which performs all the build steps in [Installing modules manually]({{site.baseurl}}/docs/developer-guide/building-modules) on a central build server and then serves your requested bundles directly to your user's browser.
+This is particularly useful for:
 
-This is especially useful for bootstrapping early stage prototypes as well as building hacks, experiments, and adding components to existing sites that weren't built with Origami in mind.  The service offers high availability, reliability, HTTPS with the same hostname and path, and its own CDN cache layer, so can be used for client-side requests.
+- early stage prototypes
+- building hacks and experiments
+- adding components to existing sites that weren't built with Origami in mind
+
+The Build Service hosts its own API and technical documentation at [build.origami.ft.com](http://build.origami.ft.com). For a step by step tutorial on how to use the Build Service, keep reading!
 
-The build service hosts its own API and technical documentation at [build.origami.ft.com](http://build.origami.ft.com).
 
 ## Building a page
 
-The following steps are a brief tutorial to get you to the point of having a working page made of Origami components working in your browser.  Intentionally this tutorial takes every possible shortcut to allow you to achieve this with no software other than a web browser, but it should arm you with everything you need to add components to a site or build one from scratch.
+This tutorial will take you though building a page that includes some Origami modules. By the end you'll have built a page using the Build Service with a [cuts the mustard test]({{site.baseurl}}/docs/developer-guide/modules/core-vs-enhanced-experience/) for older web browsers.
+
+## 1. Create a new JS Bin
+We're going to use a website called **JS Bin** for this tutorial. JS Bin lets you paste HTML, JavaScript and CSS into one panel and see the result in another. You could also use [Codepen](http://codepen.io/pen/?editors=1000), or run your own server locally.
 
-### Find the boilerplate and start a JS Bin
+To create a new JS Bin, open this link in a new tab: [Create new JS Bin](https://jsbin.com/?html,output).
 
-First, you need something to start from.  Origami provides a recommended skeleton of an HTML page to get you started, so first, find that and copy it to your clipboard:
+## 2. Put the boilerplate into the JS Bin
+For this tutorial, we'll use some boilerplate HTML. There are two things in the boilerplate that we won't be covering here; the Polyfill Service and a 'cuts the mustard test'.
```

Comment:
it does, but the boilerplate isn't deployed yet, so it's not showing. 

It's included at line 51 as a techdocs gist